app/assets/stylesheets/helpers/_body-with-contents-list.scss

```diff
@@ -0,0 +1,41 @@
+@mixin body-with-contents-list {
+  .body-with-contents-list {
+    position: relative;
+    margin-bottom: $gutter * 1.5;
+  }
+}
+
+.contents-list-link-wrapper .app-c-back-to-top {
+  margin: $gutter-half 0;
+}
+
+// sticky
+.body-with-contents-list .app-c-contents-list {
+  z-index: 0;
+}
+
+.contents-list-link-wrapper.govuk-sticky-element--stuck-to-window {
```

Comment:
This feels specific to contents list but it's styling the back to top link, should this be a component?